test/test_euler.py

```diff
@@ -707,11 +667,11 @@ def test_multilump_rhs(actx_factory, dim, order, v0):
         boundaries = {BTAG_ALL: PrescribedBoundary(lump)}
 
         inviscid_rhs = euler_operator(
-            discr, eos=IdealSingleGas(), boundaries=boundaries, q=lump_soln, t=0.0)
-        expected_rhs = lump.exact_rhs(discr, lump_soln, 0)
+            discr, eos=IdealSingleGas(), boundaries=boundaries, cv=lump_soln, t=0.0)
+        expected_rhs = lump.exact_rhs(discr, cv=lump_soln, t=0)
         print(f"inviscid_rhs = {inviscid_rhs}")
         print(f"expected_rhs = {expected_rhs}")
-        err_max = discr.norm(inviscid_rhs-expected_rhs, np.inf)
+        err_max = discr.norm((inviscid_rhs-expected_rhs).join(), np.inf)
```

Comment:
https://github.com/inducer/grudge/issues/97#issuecomment-858169109 (OK to resolve when seen)